lib/matplotlib/mpl-data/stylelib/classic.mplstyle

```diff
@@ -212,6 +212,13 @@ axes.spines.top     : True
 polaraxes.grid      : True    # display grid on polar axes
 axes3d.grid         : True    # display grid on 3d axes
 
+date.autoformatter.year   :'%Y'
+date.autoformatter.month  :'%b %Y'
+date.autoformatter.day    :'%b %d %Y'
+date.autoformatter.hour   :'%H:%M:%S'
+date.autoformatter.minute :'%H:%M:%S.%f'
+date.autoformatter.second :'%H:%M:%S.%f'
```

Comment:
Do we really need to include hours when resolving down to the fractions of a second?